v2/cf-nodejs-logging-support/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "cf-nodejs-logging-support",
+  "version": "7.0.0",
+  "description": "Logging tool for Cloud Foundry",
+  "keywords": [
+    "logging",
+    "cf",
+    "perfx"
+  ],
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/cf-nodejs-logging-support"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Federico Romagnoli, Christian Dinse"
+  },
+  "homepage": "https://sap.github.io/cf-nodejs-logging-support/",
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "fix:lint": "eslint src --ext .ts --fix",
+    "test": "mocha 'src/test/**/*.js'",
+    "test:lint": "eslint src --ext .ts"
+  },
+  "engines": {
+    "node": ">=10"
+  },
+  "devDependencies": {
+    "@types/node": "^17.0.21",
+    "@typescript-eslint/eslint-plugin": "^4.0.1",
+    "@typescript-eslint/parser": "^4.0.1",
+    "cf-nodejs-logging-support": "latest",
```

Comment:
The library cannot reference itself as dependency. Please update the test files to consume the new source for testing and remove this dependency.

Reply:
Done.